flow360/component/meshing/params.py

```diff
@@ -265,3 +264,11 @@ def flow360_json(self) -> str:
         """
 
         return self.json(encoder=flow360_json_encoder)
+
+class VolumeBeta(Volume):
```

Comment:
Can we use the flags.internal_features or advanced_featurs over here? similar to the beta_features we had before?